pom.xml

```diff
@@ -31,6 +31,16 @@
             <version>2.3.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.6.15.Final</version>
```

Comment:
also, let's use the latest version "6.2.7.Final"

Reply:
Why?